### DISCUSSION TITLE
Can't open binary on Ubuntu.

### DISCUSSION THREAD

**Discussion by mikistrange** [samuel-lucas6/Kryptor#69] [General]
Hi
This is probably down to me but I can't run the Linux x64 binary. I have changed permission to run as an executable. The right click 'Run' option does nothing and the ./kryptor command in the terminal (I am in the correct folder) does nothing. Am I missing something? I'd like to try this software so any help would be appreciated.
Thanks
Miki

**Comment by samuel-lucas6** [comment]
Hi Miki, please can you try `./kryptor -h`. You're not specifying an option by only doing `/.kryptor`. The list of options can be found [here](https://www.kryptor.co.uk/usage), and there's a [tutorial](https://www.kryptor.co.uk/tutorial) on the website.

**Comment by mikistrange** [comment]
Hi Samuel
I thought the kryptor file had to be installed but apparently not.
I've moved a copy of the file to my home / bin folder so that I can invoke
Kryptor from the command line without having to be in the extracted folder.
Everything is working now.
Thanks for your help.
Miki

On Sun, 5 Nov 2023 at 13:29, Samuel Lucas ***@***.***> wrote:

> Hi Miki, please can you try ./kryptor -h. You're not specifying an option
> by only doing /.kryptor. The list of options can be found here
> <https://www.kryptor.co.uk/usage>, and there's a tutorial
> <https://www.kryptor.co.uk/tutorial> on the website.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/samuel-lucas6/Kryptor/discussions/69#discussioncomment-7479195>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIOB6ACOG2C254W25V3NFTDYC6IDXAVCNFSM6AAAAAA66JY4W2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINZZGE4TK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
www.mikistrange.com
